apps/dashboard/src/index.css

```diff
@@ -49,33 +90,253 @@
 
-    --success: 148 72% 44%;
+    --pink-950: 330 70% 24%;
+    --pink-900: 330 71% 32%;
+    --pink-800: 330 70% 40%;
+    --pink-700: 330 70% 48%;
+    --pink-600: 330 80% 56%;
+    --pink-500: 330 96% 64%;
+    --pink-400: 330 96% 64% / 80%;
+    --pink-300: 330 96% 64% / 60%;
+    --pink-200: 330 96% 64% / 40%;
+    --pink-100: 330 96% 64% / 20%;
+    --pink-50: 330 96% 64% / 10%;
 
-    --warning: 20 100% 64%;
+    --pink-alpha-24: 330 96% 64% / 24%;
+    --pink-alpha-16: 330 96% 64% / 16%;
+    --pink-alpha-10: 330 96% 64% / 10%;
 
-    --feature: 256 88% 64%;
+    --teal-950: 172 73% 16%;
+    --teal-900: 172 64% 24%;
+    --teal-800: 172 64% 28%;
+    --teal-700: 172 72% 32%;
+    --teal-600: 172 72% 40%;
+    --teal-500: 172 72% 48%;
+    --teal-400: 172 72% 48% / 80%;
+    --teal-300: 172 72% 48% / 60%;
+    --teal-200: 172 72% 48% / 40%;
+    --teal-100: 172 72% 48% / 20%;
+    --teal-50: 172 72% 48% / 10%;
 
-    --information: 228 100% 60%;
+    --teal-alpha-24: 172 72% 48% / 24%;
+    --teal-alpha-16: 172 72% 48% / 16%;
+    --teal-alpha-10: 172 72% 48% / 10%;
 
-    --highlighted: 330 96% 64%;
+    --white-alpha-24: 0 0% 100% / 24%;
+    --white-alpha-16: 0 0% 100% / 16%;
+    --white-alpha-10: 0 0% 100% / 10%;
 
-    --stable: 172 72% 48%;
+    --black-alpha-24: 222 32% 8% / 24%;
+    --black-alpha-16: 222 32% 8% / 16%;
+    --black-alpha-10: 222 32% 8% / 10%;
 
-    --alert: 42 92% 54%;
+    /* Misc */
 
-    --verified: 200 100% 64%;
+    --static-black: var(--neutral-950);
+    --static-white: var(--neutral-0);
 
-    --input: 220 18% 90%;
-    --ring: 346 73% 50%;
+    --bg-strong: var(--neutral-950);
+    --bg-surface: var(--neutral-800);
+    --bg-sub: var(--neutral-300);
+    --bg-soft: var(--neutral-200);
+    --bg-weak: var(--neutral-50);
+    --bg-white: var(--neutral-0);
+
+    --text-strong: var(--neutral-950);
+    --text-sub: var(--neutral-600);
+    --text-soft: var(--neutral-400);
+    --text-disabled: var(--neutral-300);
+    --text-white: var(--neutral-0);
+
+    --icon-strong: var(--neutral-950);
+    --icon-sub: var(--neutral-600);
+    --icon-soft: var(--neutral-400);
+    --icon-disabled: var(--neutral-300);
+    --icon-white: var(--neutral-0);
+
+    --stroke-strong: var(--neutral-950);
+    --stroke-sub: var(--neutral-300);
+    --stroke-soft: var(--neutral-200);
+    --stroke-white: var(--neutral-0);
+
+    --faded-dark: var(--neutral-800);
+    --faded-base: var(--neutral-500);
+    --faded-light: var(--neutral-200);
+    --faded-lighter: var(--neutral-100);
+
+    --information: var(--blue-500);
+    --information-dark: var(--blue-950);
+    --information-base: var(--blue-500);
+    --information-light: var(--blue-200);
+    --information-lighter: var(--blue-50);
+
+    --warning: var(--orange-500);
+    --warning-dark: var(--orange-950);
+    --warning-base: var(--orange-500);
+    --warning-light: var(--orange-200);
+    --warning-lighter: var(--orange-50);
+
+    --error-dark: var(--red-950);
```

Comment:
Does Align UI suggest this practice? I am not well-versed in how modern design systems prefer to do this, but I'd avoid overloading CSS with too many semantics. As a pattern, it makes sense when the color needs to be reused in two semantic tokens, for example, error and destructive in our case, but I'd resolve this in the component layer.

That is, on the CSS level, we can define a red var, and then we will define a message that is an error and uses red or a destructive button that also uses red.

Let me know if this sounds reasonable.

Reply:
Yes it is how align works, and destructive actually doesn't exists in align. They just call it error which uses the red color palette. I kept here the current destructive naming during the migration period. It will be removed in favor of the align tokens for errors